src/rust/engine/engine_cffi/src/lib.rs

```diff
@@ -535,6 +535,10 @@ fn workunit_to_py_value(workunit: &Workunit) -> Option<Value> {
       externs::store_utf8("span_id"),
       externs::store_utf8(&workunit.span_id),
     ),
+    (
+      externs::store_utf8("log_level"),
```

Comment:
Calling this a "log" level is confusing, I think. Yes, that's how it's implemented, but as we've been discussing it, it's "the workunit's `level`". So probably just `level`.